packages/@aws-cdk/aws-ec2-alpha/lib/vpc-v2.ts

```diff
@@ -182,6 +191,62 @@ export interface VpcV2Props {
   readonly vpcName?: string;
 }
 
+/**
+ * Options to import a VPC created outside of CDK stack
+ */
+export interface VpcV2Attributes {
+
+  /**
+   * The VPC ID
+   * Refers to physical Id of the resource
+   */
+  readonly vpcId: string;
+
+  /**
+   * Arn of the VPC
+   * will be used to set value for account and region
+   * which then later can be used for establishing VPC peering connection
+   * @default - constructed with stack account and region value
+   */
+  readonly vpcArn?: string;
+
+  /**
+   * Primary VPC CIDR Block of the imported VPC
+   * Can only be IPv4
+   */
+  readonly vpcCidrBlock: string;
+
+  /**
+   * A VPN Gateway is attached to the VPC
+   * @default - No VPN Gateway
+   */
+  readonly vpnGatewayId?: string;
+
+  /**
+   * Public subnets associated with VPC
+   * @default -  no public subnets provided
+   */
+  readonly publicSubnets?: SubnetV2Attributes[];
+
+  /**
+   * Private subnets associated with VPC
+   * @default - no private subnets provided
+   */
+  readonly privateSubnets?: SubnetV2Attributes[];
+
+  /**
+   * Isolated subnets associated with VPC
+   * @default - no isolated subnets provided
+   */
+  readonly isolatedSubnets?: SubnetV2Attributes[];
```

Comment:
I left this comment in the readme, but I will leave it here again. I think it is better if we allow the customer to add all subnets in one attribute, and then internally we split them.